### PR TITLE
Cult Base Away Site

### PR DESCRIPTION
**Pull request by DreamySkrell**
image

changes:
  - rscadd: "Cult Base away site."
  - rscadd: "Adds some mapped in cult runes."
  - rscadd: "Adds ooc welcome message, separate from normal welcome message."
  - rscadd: "Allows corpses to use outfits."
  - rscadd: "Allows corpses use different species."
  - rscadd: "Adds some generic outfits."
  - rscadd: "Ghostspawner spawn points are actually randomly picked."
  - bugfix: "Fixes marker layers."

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!wip

**Comment by DreamySkrell** [comment]
!review

File: code/game/gamemodes/cult/runes/armor.dm [hunk 1]
```diff
@@ -51,3 +51,6 @@
 		if("Harvester")
 			to_chat(construct, SPAN_CULT("You are playing a Harvester. You are gifted with the ability to open doors with your mind, to draw runes at will, and to teleport back to Nar'Sie. Seek out all non-believers and bring them to the Geometer."))
 	to_chat(construct, SPAN_CULT("You are still bound to serve your creator, follow their orders and help them complete their goals at all costs."))
+
+/obj/effect/rune/armor/Initialize(mapload)
+	. = ..(mapload, SScult.runes_by_name[/datum/rune/armor::name])
```
[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/game/gamemodes/cult/runes/armor.dm:56]: I tried to use initial() but I couldn't make it work, but now I don't remember why
iirc you need a var for initial() so it was a bit weird here

but also according to docs this is correct way to get initial value of a var of a type

SScult is initialized here, it works fine in game, and the runes work correctly and are added to global lists

File: code/controllers/subsystems/ghostroles.dm [hunk 2]
```diff
@@ -92,6 +94,9 @@ SUBSYSTEM_DEF(ghostroles)
 	if(!length(spawnpoints[identifier])) //If we have no spawnpoints for that identifier, return false
 		return null
 
+	// if(use)
+	// 	spawnpoints[identifier] = shuffle(spawnpoints[identifier])
```
[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/controllers/subsystems/ghostroles.dm]: yeah forgot to remove

**Review by Alberyk — APPROVED** [review]

**Review by Eyeveri — APPROVED** [review]

**Pull request merged**